src/src/WebServer/NotificationPage.cpp

```diff
@@ -73,7 +73,7 @@ void handle_notifications() {
             NPlugin_ptr[NotificationProtocolIndex](NPlugin::Function::NPLUGIN_WEBFORM_SAVE, 0, dummyString);
           }
           NotificationSettings.Port                       = getFormItemInt(F("port"), 0);
-          NotificationSettings.Timeout                    = getFormItemInt(F("timeout"), NPLUGIN_001_DEF_TM);
+          NotificationSettings.Timeout                    = getFormItemInt(F("timeout"), NPLUGIN_001_DEF_TM/1000);
```

Comment:
Timeout settings for the user should be in msec, so why not also store them as msec?

Reply:
Because that means the user interface will be presented in millisecs, which is silly. We only care about secs. Converting to secs provides more convenient (faster) data input on the settings control if you use the button to change the values.

Reply:
> I have changed the system variable to store mS. Will convert to secs at the user interface. Just works nicer than mS.

I really hope this isn't needed for all, as timeouts in seconds is not something we should be waiting for in ESPEasy.
If so, then we should later look into this how it can be done by scheduling and checking via periodical checks to see if it is ready.

And like I said, all timeouts you can configure in ESPEasy are set in msec.

Reply:
This was previously fixed at 5000mS. Which is not long enough. So the timeout can now be adjusted to allow sluggish server/internet connections. Keep in mind that this is a maximum time; fast SMTP responses will finish much sooner. 

BTW, best case scenarios required about 4 secs. But I have observed Notification emails that took 15-20 secs to complete, start to finish.

I like the idea of scheduling the SMTP communications to eliminate the need for a timeout setting. But that is beyond my pay grade. 

I know some friendly devs that have the skills to do this, but they tend to be very busy. Maybe at a later time they can assist with this. I'll ask when I have coffee with one of them. :)

- Thomas